src/webkit/wkBrowser.ts

```diff
@@ -108,27 +110,17 @@ export class WKBrowser extends platform.EventEmitter implements Browser {
       // lifecycle events.
       context = this._contexts.get(pageProxyInfo.browserContextId);
     }
+    if (!context && !this._attachToDefaultContext)
+      return;
     if (!context)
       context =  this._defaultContext;
     const pageProxySession = new WKSession(this._connection, pageProxyId, `The page has been closed.`, (message: any) => {
       this._connection.rawSend({ ...message, pageProxyId });
     });
-    const pageProxy = new WKPageProxy(pageProxySession, context, () => {
-      if (!pageProxyInfo.openerId)
-        return null;
-      const opener = this._pageProxies.get(pageProxyInfo.openerId);
```

Comment:
The idea was to not create a strong reference to the opener but it should not be a big issue I think.

Reply:
Yeah, I think it's fine.